src/core/control/tools/EditSelection.cpp

```diff
@@ -75,12 +75,16 @@ EditSelection::EditSelection(UndoRedoHandler* undo, Selection* selection, XojPag
         order.insert(std::upper_bound(order.begin(), order.end(), i, EditSelectionContents::insertOrderCmp), i);
     }
 
+    Range range;
+
     for (const auto& [e, i]: order) {
+        range.addPoint(e->getX(), e->getY());
+        range.addPoint(e->getX() + e->getElementWidth(), e->getY() + e->getElementHeight());
```

Comment:
The call to `calcSizeFromElements` at the beginning of the constructor suggests the bounding box has already been computed. Can't we use that instead of recomputing it?